packages/embark-ui/src/containers/TransactionsContainer.js

```diff
@@ -81,33 +102,29 @@ class TransactionsContainer extends Component {
     const estNumberOfTxs = this.estNumberOfTxs;
     return txs.filter((tx, idx) => {
       const txNumber = estNumberOfTxs - idx;
-      const index = (
-        (estNumberOfTxs -
-         (this.numTxsToDisplay * (this.state.currentPage - 1))) -
-          txNumber + 1
-      );
+      const index =
+        estNumberOfTxs -
+        this.numTxsToDisplay * (this.state.currentPage - 1) -
+        txNumber +
+        1;
```

Comment:
Can we get the `+1 ` on the line above again?

Reply:
prettier did it when I had it auto-format that section the code, but I can bump it up and not rerun prettier on that section.

Reply:
Done.